cbrain_task/civet_macaque/bourreau/civet_macaque.rb

```diff
@@ -466,7 +470,7 @@ def save_results
       self.addlog("Error: it seems this CIVET run is still processing!")
       self.addlog("We found these files in 'logs' : #{running.sort.join(', ')}")
       self.addlog("Trigger the recovery code to force a cleanup and a try again.")
-      return false # Failed On Cluster
+      is_partial = true
```

Comment:
restore the original code here, when the CIVET is still processing we want it to Fail On Cluster so that a restart is possible by the user. We don't need to save a partial result here